pkgs/applications/window-managers/openbox/default.nix

```diff
@@ -7,9 +8,13 @@ stdenv.mkDerivation rec {
   version = "3.6.1";
 
   nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
```

Comment:
Have you tried `autoreconfHook` instead?

Reply:
Tried it before, but now it works. Not that I've added `python3` to nativeBuildInputs as well to allow the autoreconf to find the python interpreter to set.